specificatie/partner.yaml

```diff
@@ -25,6 +25,8 @@ components:
           $ref: '#/components/schemas/GbaAangaanHuwelijkPartnerschap'
         ontbindingHuwelijkPartnerschap:
           $ref: "#/components/schemas/GbaOntbindingHuwelijkPartnerschap"
+        rni:
```

Comment:
In de wijziging van het LO is voor partnerhistorie geen rni opgenomen. Is ook logisch want in RNI worden geen partnergegevens bijgehouden. Moet er denk ik uit